### ISSUE TITLE
[proposal] add size prop to ListItem icons' configuration options

### ISSUE THREAD

**Issue by raduflp** [react-native-training/react-native-elements#812]
Currently the size of the left/right icon in ListItem can be modified only through `{ style: {fontSize} }`
Personally I think it would be more intuitive to add a `size` configuration that would adjust the Icon size prop. I think it's a common enough scenario that would justify the change.
Happy to make a pull request if we agree on that.

**Comment by xavier-villelegier** [comment]
Hey @raduflp!
Thanks a lot for your recent activities on this repo, it really helps to make it better and better 👍

Actually, you already can haha 😅

```jsx
 <ListItem
  title="List item"
  leftIcon={{
    name:'activity',
    type:'feather',
    size: 30
  }}
/>
```

Check out [this example](https://snack.expo.io/rk7X5xEVG) 👌

But the implementation isn't that clean. If you look at [this part](https://github.com/react-native-training/react-native-elements/blob/master/src/list/ListItem.js#L128) we should spread the props instead of injecting them one by one, to anticipate eventual new future props of the `Icon`

**Comment by raduflp** [comment]
Hey @xavier-villelegier! 
you are right, it works for the `leftIcon`. I was having the issues with the `rightIcon` because the `size` prop there is hardcoded to `28` and the icon looks way too big. I just assumed it is the same implementation as `leftIcon`, so a small adjustment to the `rightIcon Icon.size` would fix it for now. 
I totally agree that spreading the props would be much cleaner, although prop names for `Icon.iconStyle` and `IconObject.style` mismatch.

**Comment by xavier-villelegier** [comment]
@raduflp Ahhh my bad, you're right about the `rightIcon`. Feel free to submit a PR by using `rightIcon.size || 28` then 👍

**Comment by k3ithl1m** [comment]
Can I work on this enhancement?

**Comment by Monte9** [comment]
@k3ithl1m this has been fixed in this PR: https://github.com/react-native-training/react-native-elements/pull/852 and will be release in `beta-2` today

**Comment by xavier-villelegier** [comment]
Thanks again for the proposal @raduflp, this is released in `v1.0.0-beta2` 👍
